### ISSUE TITLE
Bug: RainBird controller request failed.

### ISSUE THREAD

**Issue by tbaur** [homebridge-plugins/homebridge-rainbird#390]
### Describe The Bug

1.8.0 TypeError

### To Reproduce

Upgrade to 1.8.0

### Expected behavior

N/A

### Relevant log output

```shell
[10/19/2022, 11:22:53 AM] [RainBird] RainBird controller request failed. [TypeError: Cannot read properties of undefined (reading 'post')]
[10/19/2022, 11:22:53 AM] [RainBird] Will retry in 60 seconds
[10/19/2022, 11:23:53 AM] [RainBird] RainBird controller request failed. [TypeError: Cannot read properties of undefined (reading 'post')]
[10/19/2022, 11:23:53 AM] [RainBird] Will retry in 60 seconds
[10/19/2022, 11:24:53 AM] [RainBird] RainBird controller request failed. [TypeError: Cannot read properties of undefined (reading 'post')]
[10/19/2022, 11:24:53 AM] [RainBird] Will retry in 60 seconds
[10/19/2022, 11:25:53 AM] [RainBird] RainBird controller request failed. [TypeError: Cannot read properties of undefined (reading 'post')]
[10/19/2022, 11:25:53 AM] [RainBird] Will retry in 60 seconds
[10/19/2022, 11:26:53 AM] [RainBird] RainBird controller request failed. [TypeError: Cannot read properties of undefined (reading 'post')]
[10/19/2022, 11:26:53 AM] [RainBird] Will retry in 60 seconds
[10/19/2022, 11:27:53 AM] [RainBird] RainBird controller request failed. [TypeError: Cannot read properties of undefined (reading 'post')]
[10/19/2022, 11:27:53 AM] [RainBird] Will retry in 60 seconds
[10/19/2022, 11:28:53 AM] [RainBird] RainBird controller request failed. [TypeError: Cannot read properties of undefined (reading 'post')]
[10/19/2022, 11:28:53 AM] [RainBird] Will retry in 60 seconds
[10/19/2022, 11:29:53 AM] [RainBird] RainBird controller request failed. [TypeError: Cannot read properties of undefined (reading 'post')]
[10/19/2022, 11:29:53 AM] [RainBird] Will retry in 60 seconds
```

### Config for homebridge-rainbird

```shell
N/A
```

### Device & Model

N/A

### Node.js Version

16.18.0

### NPM Version

8.19.2

### Homebridge Version

1.5.0

### Homebridge Rainbird Plugin Version

1.8.0

### Operating System

macOS

**Comment by donavanbecker** [comment]
Can you try the latest beta: 

[![npm version](https://badgen.net/badge/homebridge-rainbird@beta/>=1.8.1-beta.2/orange)](https://www.npmjs.com/package/homebridge-rainbird)

[Click here to find out how to install Beta](https://github.com/donavanbecker/homebridge-rainbird/wiki/Beta-Version)

**Comment by donavanbecker** [comment]
Fixed in v1.8.1
